Adina-Jaffee-semantic.html

```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+   <head>
+      <meta charset="utf-8">
+      <title>Betamore FEWD Course | Accessibility Exercise</title>
+      <style>
+         #submit {
+         display: inline-block;
+         border: 2px solid #333;
+         border-radius: 5px;
+         padding: 5px 10px;
+         }
+      </style>
+   </head>
+   <body>
+      <h1>
+         <font size="16">My Main Heading</font>
```

Comment:
Let's get rid of the `font` elements and associated attributes from our markup. See more info on why we should remove it here: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/font